### ISSUE TITLE
Improve search algorithm

### ISSUE THREAD

**Issue by ivan-kleshnin** [Widdershin/cycle-ecosystem#5]
[davidmarkclements/react-functional](https://github.com/davidmarkclements/react-functional) is irrelevant. It just contains word "cycle". While I can make a PR to add it to irrelevant list you should probably look into `package.json` for `dependencies` and `peerDependencies` instead of description text. Otherwise the list will be inevitably flooded with irrelevant stuff.

**Comment by Widdershin** [comment]
Agreed. There's an issue open to implement dependency inspection, #3. If you're interested in making a pull request, that would be awesome!

**Comment by ivan-kleshnin** [comment]
@Widdershin sure. I will give it look to the end of the week.
